torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -264,109 +280,101 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                     new_node = g.call_function(
                         ttnn.full, args=(tuple(args[0]),), kwargs=new_kwargs
                     )
-                    new_node.meta["val"] = node.meta["val"]
-                    node.replace_all_uses_with(
-                        new_node,
-                        delete_user_cb=lambda node: node != new_node,
-                    )
+                    new_nodes.append(new_node)
+                else:
+                    # Replace op with scalar for eltwise ops
+                    # TODO: Generalize this to support all eltwise ops
+                    node_users = list(node.users.keys())
+                    for node_user in node_users:
+                        if node_user.target == torch.ops.aten.div.Tensor:
+                            node_user.update_arg(1, args[1])
             if node.target == torch.ops.aten.baddbmm.default:
                 kwargs = node.kwargs
                 # out = beta * input + alpha * (batch1 @ batch2)
                 # if beta is 0, input is ignored, and nan and inf in it will not be propogated
                 new_node = g.call_function(ttnn.matmul, args=(args[1], args[2]))
-                # Intermediate node meta is not preserved, this ensures retention
-                new_node.meta["val"] = node.meta["val"]
```

Comment:
I thought we have to carry over meta["val], no?

Reply:
So same idea as https://github.com/tenstorrent/pytorch2.0_ttnn/pull/53#discussion_r1717567654. `g.call_function` will now carry over the metadata of the current node automatically.